backend/src/main/resources/db/changelog/db.changelog-v4.yaml

```diff
@@ -0,0 +1,72 @@
+###############################################################
```

Comment:
I would say you don't need to create a new file for this change. You can add these changes into db.changelog-v3.yaml file also.

Reply:
incorporated the review comment.